integration-tests/test-bundle/build.gradle

```diff
@@ -10,6 +10,7 @@ test.enabled = false
 
 dependencies {
     compileOnly enforcedPlatform("eu.xenit.alfresco:alfresco-community-bom:5.0.d")
```

Comment:
We don't need this anymore then?

Reply:
There are no dependencies added to the `compileOnly` configuration directly, so this declaration should be removed.

Reply:
Removed!